### PR TITLE
Create github-sso-recipe

### PR DESCRIPTION
**Pull request by rehabHassanein**

### CODE REVIEW

File: github-sso-recipe.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Enable SSO for Semaphore SSH key `semaphore-key`.
+2. Grant orgnization access to Semaphore 2.0 deployment app.
+
+Below you'll find detailed steps on how to accomplish both steps.
+
+## Enable SSO for Semaphore SSH key
+1. Goto https://github.com/settings/profile
+2. From the left side menu click "[SSH and GPG keys]: https://github.com/settings/keys".
+3. Find the key with id "semaphore-key".
```
Comment (markoa) [inline github-sso-recipe.md]: Shouldn't the user need to create it first?

Comment (rehabHassanein) [reply github-sso-recipe.md]: I believe this key is automatically created once you connect Semaphore to your Github.

Comment (rehabHassanein) [reply github-sso-recipe.md]: Give me some more time to test adding the repository without Enabling SSO to `semaphore-key`.

Comment (rehabHassanein) [reply github-sso-recipe.md]: It looks like this step is unnecessary! @markoa . Updating the file.

**Pull request merged**